package.json

```diff
@@ -40,11 +40,11 @@
     "@angular/platform-browser-dynamic": "^8.0.0",
     "@angular/router": "^8.0.0",
     "@asymmetrik/ngx-leaflet": "3.0.1",
-    "@nebular/auth": "4.1.0",
-    "@nebular/bootstrap": "4.1.0",
-    "@nebular/eva-icons": "4.1.0",
-    "@nebular/security": "4.1.0",
-    "@nebular/theme": "4.1.0",
+    "@nebular/auth": "4.1.2",
+    "@nebular/bootstrap": "4.1.2",
```

Comment:
since we removed almost completely, do we need it here?

Reply:
You right, removed in e7f91b8